lib/pushmeup/apnsv3/notification.rb

```diff
@@ -35,11 +39,11 @@ def body
 
     def to_hash
       aps = {}
-
       aps.merge!(alert: alert) if alert
       aps.merge!(badge: badge) if badge
       aps.merge!(sound: sound) if sound
       aps.merge!(category: category) if category
+      aps.merge!(other: other) if other
```

Comment:
This is how it is added in here https://github.com/AylaNetworks/pushmeup/blob/master/lib/pushmeup/apns/notification.rb

      aps.merge!(self.other) if self.other

Looks like the format will be different